src/applications/simple-forms-form-engine/shared/utils/formConfig.js

```diff
@@ -45,21 +45,33 @@ const formatChapters = chapters =>
     {},
   );
 
+export const statementOfTruthBody =
+  'I confirm that the identifying information in this form is accurate and ' +
+  'has been represented correctly.';
+
+/** @returns {FormConfig} */
 export const createFormConfig = (form, options) => {
   const { chapters, formId, ombInfo, title } = form;
   const { rootUrl, trackingPrefix } = options;
   const subTitle = `VA Form ${formId}`;
 
   return {
+    preSubmitInfo: {
+      statementOfTruth: {
+        body: statementOfTruthBody,
+        messageAriaDescribedby: statementOfTruthBody,
+        fullNamePath: 'fullName',
+      },
+    },
     rootUrl,
     urlPrefix: '/',
-    trackingPrefix,
     // eslint-disable-next-line no-console
     submit: () => console.log(`Submitted ${subTitle}`),
     introduction: props => <IntroductionPage {...props} ombInfo={ombInfo} />,
     confirmation: ConfirmationPage,
     formId,
     saveInProgress: {},
+    trackingPrefix,
```

Comment:
Only moved this to make the keys slightly more alphabetical. It can be moved back if desired.

Reply:
I don't have strong feelings. My initial thought is that these are not in alphabetical order, but rather some other logical grouping, and in that sense `trackingPrefix` feels like it belongs where it was (next to `urlPrefix`, `rootUrl` and other basic config-type properties). Some of the stuff below that feels more "code" to me (e.g. submit handler, and Intro/Confirmation components). But there are other pretty basic config-type props below those, so 🤷‍♂️. 

Did we arrive at this initially because we were following other convention (i.e. existing form configs)? Does that matter?

I'm not concerned at all with this change. Thanks for the comment here.